evap/staff/tests/test_views.py

```diff
@@ -1263,6 +1263,20 @@ def test_last_modified_unchanged(self):
         self.assertEqual(self.evaluation.last_modified_user, last_modified_user_before)
         self.assertEqual(self.evaluation.last_modified_time, last_modified_time_before)
 
+    def test_approval(self):
+        """
+            Tests if approving the evaluation changes the state and updates the manager approval field
+        """
+        self.assertIsNone(self.evaluation.manager_approval_user)
+
+        page = self.app.get(self.url, user=self.user, status=200)
+        form = page.forms["evaluation-form"]
+        form.submit(name="operation", value="approve")
+
+        self.evaluation = Evaluation.objects.get(pk=self.evaluation.pk)
```

Comment:
I'd prefer `self.evaluation.reload_from_db()` here (for me it's faster to understand what's happening)

Reply:
might be a problem: https://github.com/fsr-de/EvaP/blob/9a268798a55c9e15b9ffaed6158a33ffc628fc54/evap/evaluation/tests/test_models.py#L378-L380

Reply:
hmm interesting. Someone will have to explain to me why exactly it does not work next time we need, but the code is fine for me then.

Reply:
it does not work because of the `state` field which is an `FSMField(...., protected=True)`, which forbids any direct (i.e. not using the transitions) modification of the state field. refresh_from_db would perform such a direct modification.

we could probably write our own reload_from_db for that, but it might not be worth it :)